Code/src/main/java/nl/utwente/group10/ui/components/ConnectionLine.java

```diff
@@ -0,0 +1,65 @@
+package nl.utwente.group10.ui.components;
+
+import java.io.IOException;
+
+import nl.utwente.ewi.caes.tactilefx.control.TactilePane;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.shape.CubicCurve;
+
+
+/**
+ * This class represent a Connection-Line visual object in the UI.
+ * Each UI element that uses Connection-Line properties should extend
+ * from here.
+ * 
+ * For Lines based on Start and End anchors, see AnchoredConnectionLine
+ * For Lines that connect inputs and outputs of Blocks see Connection.
+ */
+public class ConnectionLine extends CubicCurve {
+	/** Control offset for this bezier of this line */
```

Comment:
Dit is de bochtigheid van de lijn, toch, in feite? Zo ja kun je dat er misschien bij zetten.